pkg/web/cookie/cookie.go

```diff
@@ -52,7 +52,7 @@ func (d *Cookie) new(r *http.Request) http.Cookie {
 	return http.Cookie{
 		Name:     d.Name,
 		Path:     d.Path,
-		MaxAge:   int(d.MaxAge.Nanoseconds() / 1000),
+		MaxAge:   int(d.MaxAge.Seconds()),
```

Comment:
Oh seriously? This has been here since 2018?
https://github.com/TheThingsNetwork/lorawan-stack/commit/9b3570a9cd25599b8d0f6999ae3c32b9a014f83d

Reply:
Yes... Only affected the state cookie though, so not really dramatic.